share/tasks.py

```diff
@@ -199,3 +201,31 @@ def schedule_harvests(self, *source_config_ids, cutoff=None):
             logs.extend(HarvestScheduler(source_config).all(cutoff=cutoff, save=False))
 
         HarvestLog.objects.bulk_get_or_create(logs)
+
+
+@celery.shared_task(bind=True)
+def source_stats(self):
+    oai_sourceconfigs = SourceConfig.objects.filter(
+        disabled=False,
+        base_url__isnull=False,
+        harvester__key='oai'
+    )
+    for config in oai_sourceconfigs.values():
+        get_source_stats.apply_async((True, config['id']))
+
+    non_oai_sourceconfigs = SourceConfig.objects.filter(
+        disabled=False,
+        base_url__isnull=False
+    ).exclude(
+        harvester__key='oai'
+    )
+    for config in non_oai_sourceconfigs.values():
+        get_source_stats.apply_async((False, config['id']))
+
+
+@celery.shared_task(bind=True)
+def get_source_stats(self, oai, config_id):
+    if oai:
```

Comment:
Would it make more sense to just load the config here and just check if it's oai?